.storybook/main.ts

```diff
@@ -0,0 +1,38 @@
+// Imports the Storybook's configuration API
```

Comment:
Derived from https://storybook.js.org/docs/react/configure/overview#using-storybook-api and https://github.com/storybookjs/storybook/issues/17589